extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyCommonProcessor.java

```diff
@@ -62,8 +62,8 @@
          * If the limit is exceeded, Resteasy will return Response
          * with status 413("Request Entity Too Large")
          */
-        @ConfigItem
-        public OptionalInt maxInput;
+        @ConfigItem(defaultValue = "10M")
```

Comment:
Out of curiosity, what lead you to picking 10meg as the default? I am not necessarily arguing for something different, I am just wondering if its mirroring a convention from elsewhere.

Reply:
I lifted it out of Resteasy. :-)